src/js/game/systems/item_ejector.js

```diff
@@ -294,7 +300,18 @@ export class ItemEjectorSystem extends GameSystemWithFilter {
             }
         }
 
-        return false;
+        // it could be a custom mod case
+        // let's check if it is any of them
+        var any = false; // so that all matching components can process it
+        for (let id in MOD_ITEM_FILTERS) {
+            if (receiver.components[id]) {
+                let handler = MOD_ITEM_FILTERS[id];
+                if (handler(item, receiver, slotIndex)) any = true;
+                else return false;

```

Comment:
this makes the system pass over an item if (it would by default) _or_ (_a_ filter passes _and_ _all_ filters pass), which feels like a really awkward combination; particularly, *a* filter increases opportunities, but *more* filters decrease opportunities

Reply:
obliterated too